packages/widgetbook/test/src/widgetbook_test.dart

```diff
@@ -40,11 +40,19 @@ void main() {
             'categories is empty',
             () {
               expect(
-                () => Widgetbook(
-                  addons: const [],
+                () => Widgetbook<ThemeData>(
+                  addons: [
+                    WidgetbookAddOn(
+                      name: 'A',
+                      builder: (context) => const SizedBox(),
+                    ),
+                  ],
+                  ],
```

Comment:
The test file does not compile in general, but this syntax has one `]` too much.